nx.json

```diff
@@ -59,13 +59,8 @@
             "inputs": ["default"]
         }
     },
-    "implicitDependencies": {
```

Comment:
There must be some kind of replacement, because this is kind of important, because it resets cache for all packages if any of these files are changed.

Reply:
1a84523ae8cbbfca2dc7bac4af368d1429b73010